src/main/resources/dbreplica/migration/V1.0.9__task_assignments_plsql.sql

```diff
@@ -0,0 +1,84 @@
+--
+-- Copies a task record into the history table.
```

Comment:
Comment seems wrong.

Reply:
As discussed with Ravi, this will be refactored RWA-2085

Reply:
Fixed comments